ui/ducks/metamask/metamask.js

```diff
@@ -279,15 +284,13 @@ export function getBlockGasLimit(state) {
   return state.metamask.currentBlockGasLimit;
 }
 
-export function getConversionRate(state) {
-  return state.metamask.conversionRate;
+export function getNativeCurrency(state) {
+  return getProviderConfig(state).ticker;
```

Comment:
🙌  keeping the nativeCurrency to a single source of truth tightly coupled to the provider config makes so much more sense! Even if we are going to get rid of providerConfig (and a single globally selected native currency) soon.